src/components/Compare/CompareMain.tsx

```diff
@@ -30,73 +46,38 @@ const CompareMain = (props: {
   isHomepage?: boolean;
   currentCounty?: any;
   stateId?: string;
-  autoScroll?: boolean;
 }) => {
   const tableRef = useRef<HTMLDivElement>(null);
-  const scrollOffset = props.isHomepage ? 75 : 165;
-  const scrollTo = (div: null | HTMLDivElement) =>
-    div &&
-    window.scrollTo({
-      left: 0,
-      top: div.offsetTop - scrollOffset,
-      behavior: 'smooth',
-    });
+  const scrollToCompare = useCallback(() => {
+    const scrollOffset = props.isHomepage ? 75 : 165;
+    // Note (Chelsi): short delay is needed to make scrollTo work
+    return setTimeout(() => {
+      if (tableRef.current) {
+        window.scrollTo({
+          left: 0,
+          top: tableRef.current.offsetTop - scrollOffset,
+          behavior: 'smooth',
+        });
+      }
+    }, 250);
+  }, [props.isHomepage, tableRef]);
 
   const location = useLocation();
-  const queryParams = QueryString.parse(location.search, {
```

Comment:
I ripped out all the query param stuff in favor of the Firestore stuff.  😢  We can bring it back if we need / want it in the future.